### PR TITLE
Bdog 2362

### PR DESCRIPTION
**Pull request by RikSherman**

### CODE REVIEW

File: app/uk/gov/hmrc/servicedependencies/controller/ServiceDependenciesController.scala [hunk 1]
```diff
@@ -144,20 +144,31 @@ class ServiceDependenciesController @Inject()(
         .map(_.fold(NotFound(""))(res => Ok(Json.toJson(res))))
     }
 
-  def moduleDependenciesAllVersions(repositoryName: String): Action[AnyContent] =
+  def moduleDependencies(repositoryName: String, versionOption: Option[String]): Action[AnyContent] =
     Action.async {
       for {
-        meta           <- metaArtefactRepository.findAllVersions(repositoryName)
+        metaArtefacts  <- versionOption match {
+                            case Some(version) if version equalsIgnoreCase("latest") => metaArtefactRepository.find(repositoryName) map {
```
Comment (colin-lamed) [inline app/uk/gov/hmrc/servicedependencies/controller/ServiceDependenciesController.scala:151]: `.map(_.toSeq)` should tidy this up

File: app/uk/gov/hmrc/servicedependencies/controller/ServiceDependenciesController.scala [hunk 2]
```diff
@@ -206,7 +217,7 @@ class ServiceDependenciesController @Inject()(
       ).merge
     }
 
-  private def getRepositoryInfo(meta: MetaArtefact, latestVersions: Seq[LatestVersion], bobbyRules: BobbyRules) = {
+  private def toRepositoryInfo(meta: MetaArtefact, latestVersions: Seq[LatestVersion], bobbyRules: BobbyRules) = {
```
Comment (colin-lamed) [inline app/uk/gov/hmrc/servicedependencies/controller/ServiceDependenciesController.scala:220]: can we call it `toRepository` to match the type it returns?

**Review by colin-lamed — DISMISSED** [review]

File: app/uk/gov/hmrc/servicedependencies/controller/ServiceDependenciesController.scala (hunk 1 above)
Comment (RikSherman) [reply app/uk/gov/hmrc/servicedependencies/controller/ServiceDependenciesController.scala]: Done

File: app/uk/gov/hmrc/servicedependencies/controller/ServiceDependenciesController.scala (hunk 2 above)
Comment (RikSherman) [reply app/uk/gov/hmrc/servicedependencies/controller/ServiceDependenciesController.scala]: Done

**Review by colin-lamed — APPROVED** [review]

**Pull request merged**